### ISSUE TITLE
error to post image

### ISSUE THREAD

**Issue by phls** [xray7224/p#42]
Hi,

I'm trying to post an image using:
./p post image logo.png

But I received:

Traceback (most recent call last):
  File "./p", line 985, in <module>
    cli()
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 716, in **call**
    return self.main(_args, *_kwargs)
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, *_ctx.params)
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(_args, *_kwargs)
  File "/root/p-master/local/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], *_kwargs)
  File "/root/p-master/local/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(_args, *_kwargs)
  File "./p", line 425, in p_post_image
    p_post_file(p, p.pump.Image, path, title, description, license, to, cc, echovar)
  File "./p", line 386, in p_post_file
    myfile.from_file(path)
  File "/root/p-master/src/pypump/pypump/models/__init__.py", line 624, in from_file
    headers=headers,
  File "/root/p-master/src/pypump/pypump/pypump.py", line 311, in request
    *_request
  File "/root/p-master/src/pypump/pypump/pypump.py", line 354, in _requester
    response = fnc(url, *_kwargs)
  File "/root/p-master/local/lib/python2.7/site-packages/requests/sessions.py", line 522, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/root/p-master/local/lib/python2.7/site-packages/requests/sessions.py", line 461, in request
    prep = self.prepare_request(req)
  File "/root/p-master/local/lib/python2.7/site-packages/requests/sessions.py", line 394, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/root/p-master/local/lib/python2.7/site-packages/requests/models.py", line 295, in prepare
    self.prepare_headers(headers)
  File "/root/p-master/local/lib/python2.7/site-packages/requests/models.py", line 409, in prepare_headers
    check_header_validity(header)
  File "/root/p-master/local/lib/python2.7/site-packages/requests/utils.py", line 800, in check_header_validity
    "not %s" % (value, type(value)))
requests.exceptions.InvalidHeader: Header value 13652 must be of type str or bytes, not <type 'int'>

**Comment by moggers87** [comment]
Could you please tell us which version of `p` and `pypump` you're using?

**Comment by craigmaloney** [comment]
This is because Requests is no longer accepting int header sizes, and requires them to be str or bytes.

I made a pull request for PyPump to resolve: https://github.com/xray7224/PyPump/pull/154

**Comment by phls** [comment]
Hi, I tryng to package "p" to Debian, like was requested here: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=762367

So, I'm using pypump 0.7.

By the way, I found other problem:

At first I packaged to use Python 3 only. But after the package was created, when I tried to authorize my account, I received this error:
NameError: name 'unicode' is not defined

I searched I little and found this:
http://stackoverflow.com/questions/36110598/nameerror-name-unicode-is-not-defined#comment59862846_36110598
"There is no such name in Python 3, no. You are trying to run Python 2 code in Python 3. In Python 3, unicode has been renamed to str."

If this is correct, I can't package to Python 3, right?

Thnaks,
